### PR TITLE
Fix Visualization tool window goldbar and auto show/hide

### PR DESCRIPTION
**Pull request by rita-g-sonarsource**

### CODE REVIEW

File: src/IssueViz/IssueVisualizationControl/IssueVisualizationToolWindow.cs [hunk 1]
```diff
@@ -57,6 +57,9 @@ protected override void Dispose(bool disposing)
             if (disposing)
             {
                 viewModel.Dispose();
+
+                var vsWindowFrame = Frame as IVsWindowFrame;
+                vsWindowFrame.CloseFrame((uint)__FRAMECLOSE.FRAMECLOSE_NoSave);
```
Comment (duncanp-sonar) [inline src/IssueViz/IssueVisualizationControl/IssueVisualizationToolWindow.cs:62]: Presumably this is testable?

**Review by duncanp-sonar — APPROVED** [review]
The change looks ok, but why no unit test?

**Comment by rita-g-sonarsource** [comment]
> The change looks ok, but why no unit test?

I don't know what you're talking about, of course there are unit tests 😛

**Comment by duncanp-sonar** [comment]
My mistake, I wasn't wearing my "seeing into the future" glasses ;-)

**Pull request merged**